Cargo.toml

```diff
@@ -18,4 +18,5 @@ plotters-backend = "^0.3.*"
 debug = []
 
 [dev-dependencies]
-plotters = {version = "^0.3.0", default_features = false, features = ["ttf"]}
+plotters = { version = "^0.3.0", default_features = false, features = ["ttf"] }
+regex = "1.5"
```

Comment:
I would suggest the latest version here, `regex = "1.5.5"`. Thanks for your contributions.

Reply:
[The crate doc](https://crates.io/crates/regex) says it's fine with `"1.5"`. Should I change?

Reply:
Cargo [chooses the latest version](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html) of regex, so my suggestion can be innocuous, but can also work as a matter of best practice. For example, the [crate doc](https://crates.io/crates/regex) needs a manual update; the version number `1.5` should be changed to reflect [this security advisory](https://github.com/rust-lang/regex/security/advisories/GHSA-m5pq-gvj9-9vr8) at least.